game/zx_spectrum_tap.ksy

```diff
@@ -0,0 +1,52 @@
+meta:
+  id: zx_spectrum_tap
+  file-extension: tap
+  endian: le
+  license: CC0-1.0
+  title: ZX Spectrum tape file format
+  xref:
+    justsolve: TAP_(ZX_Spectrum)
+    pronom: fmt/801
+doc-ref: https://faqwiki.zxnet.co.uk/wiki/TAP_format
+seq:
+  - id: block
+    type: block
+    repeat: eos
+enums:
+  flag_enum:
+    0x00: header
+    0xFF: data
+  header_type_enum:
+    0: program
+    1: num_array
+    2: char_array
+    3: bytes
+types:
+  block:
+    seq:
+      - id: len_header
+        contents: [0x13, 0x00]
```

Comment:
Is it really meant to be hardcoded? This spec doesn't look futureproof.

Reply:
All ROM headers are 0x13 bytes long (see http://www.zx-modules.de/fileformats/tapformat.html). Does need some rework to allow for non-header data blocks.

Reply:
If they always were, there would be no need to put that number into files. This stuff is usually done if the number can change in future. I know that spectrum is an obs9pete platfom

Reply:
Resolved